### PR TITLE
feat: Use `StandardUICommand` for `TextBox` context menu

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): related to https://github.com/unoplatform/uno/issues/9417

## PR Type

What kind of change does this PR introduce?

- Feature

## What is the current behavior?

`StandardUICommands` not used

## What is the new behavior?

`StandardUICommands` used

### <samp>🤖 Generated by Copilot at 2c2b37f</samp>

Removed unused resource files for `TextBox` control and updated its context menu to use `StandardUICommand` for better localization and accessibility.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by ramezgerges** [comment]
Working on my end 👍

**Review by ramezgerges — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.skia.cs [hunk 1]
```diff
@@ -647,11 +647,11 @@ protected override void OnRightTapped(RightTappedRoutedEventArgs e)
 				_contextMenu.Opened += (_, _) => UpdateDisplaySelection();
 
 				// TODO: port localized resources from WinUI
```
Comment (ramezgerges) [inline src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.skia.cs:649]: We can also drop this TODO 😄

**Review by ajpinedam — APPROVED** [review]
👏

File: src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.skia.cs (hunk 1 above)
Comment (ramezgerges) [reply src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.skia.cs:649]: Well, let's let auto-merge do its thing. The whole context menu thing will need to be reworked after the port anyway.

**Pull request merged**